src/lib.rs

```diff
@@ -108,12 +110,16 @@ impl TargetDep {
 
             // Move output
             std::fs::rename(out_path, &self.output_path).unwrap_or_else(|err| {
```

Comment:
For this kind of thing `if let Err(err) = ...` is a little easier to follow than `unwrap_or_else`.

Reply:
thank you. changed this as well.